scripts/entrypoint.sh

```diff
@@ -18,6 +18,7 @@ GTP_EXTENSION_HEADER_PRESENT=${GTP_EXTENSION_HEADER_PRESENT:-no}
 ENABLE_5G_FEATURES=${ENABLE_5G_FEATURES:-no}
 REGISTER_NRF=${REGISTER_NRF:-no}
 USE_FQDN_NRF=${USE_FQDN_NRF:-no}
+#SPGWC0_IP_ADDRESS=${SPGWC0_IP_ADDRESS:-127.0.0.1}
```

Comment:
@orion-belt 

Can you remove this commented line?

Reply:
@rdefosse 
Done